.test-infra/metrics/grafana/dashboards/perftests_metrics/Python_IO_IT_Tests_Dataflow.json

```diff
@@ -482,6 +482,128 @@
         "align": false,
         "alignLevel": null
       }
+    },
```

Comment:
Feel free to add another reviewer for these graphana changes as I'm not really familiar enough to give good feedback.

Reply:
Thanks. This trivially adds a graph, see #23914 for some context.